containers/docker-compose.yaml

```diff
@@ -120,6 +120,27 @@ services:
       - api
       - all
 
+  gds-bff:
+    build:
+      context: ../
+      dockerfile: ./containers/bff/Dockerfile
+    image: trisa/gds-bff
+    init: true
+    depends_on:
+      - gds
```

Comment:
Will we also (at some point) need to update the container directives for `gds-ui` and `gds-admin-ui` to depend on the BFF?

Reply:
Oh good point - we'll have to remember when we add the `gds-user-ui` that it will depend on the BFF - `gds-ui` should go away; I don't think we have any plans right now for `gds-admin-ui` to depend on the BFF - but happy to discuss that.